visualization/app/codeCharta/ui/edgeSettingsPanel/edgeSettingsPanel.component.html

```diff
@@ -64,14 +64,14 @@
 <cc-map-color-picker map-color-for="outgoingEdge" label="Outgoing Edge"> </cc-map-color-picker>
 <cc-map-color-picker map-color-for="incomingEdge" label="Incoming Edge"> </cc-map-color-picker>
 
-<md-input-container class="md-block">
+<md-input-container>
 	<md-checkbox
 		class="md-primary"
 		ng-model="$ctrl._viewModel.showOnlyBuildingsWithEdges"
 		ng-change="$ctrl.applyShowOnlyBuildingsWithEdges()"
 		ng-disabled="$ctrl._viewModel.totalAffectedBuildings == 0"
 	>
-		Show only nodes with edges
+		Only nodes with edges
```

Comment:
NIT: I personally prefer the old style.

Reply:
It didnt fit in 1 line when trying to display the icon next to it, and making a line break for the icon didn't look really nice